CHANGELOG.md

```diff
@@ -24,13 +24,15 @@ Here is a template for new release sections
 ### Added
 - `E-Land_Requirements.rst`: Official E-Land requirement list as well as progress on functional and non-functional requirements (#590)
 - Add pytests for `E4.detect_excessive_excess_generation_in_bus()` (#591)
-
+- Tests for D1.transformer() (#596)
 ### Changed
 - Order of readthedocs content (#590)
 
 ### Removed
 
 ### Fixed
+- All `variable_costs`, `efficiency` and `nominal_value` of transformers on output flows (#596)
```

Comment:
Hmm, all I did is assigning `variable_costs`, `efficiency` and `nominal_value` to output flows instead of assigning some to in- and some to output flows (which was how it was done before). Could you state which explanations you would like to mention?

Reply:
You removed the part that looped thorugh `index` and `efficiencies` are now not read from a list anymore. I do agree that you did it just to assign the above, but maybe we run into errors later and it would be good to now that you changed the processing in `D1.transformer_constant_efficiency_fix()` and `D1.transformer_constant_efficiency_optimize()`. I think that also changed the tests? I really just mean a changelog entry like

> Changed `D1.transformer_constant_efficiency_fix()`  and `D1.transformer_constant_efficiency_optimize()` as well as their tests to reassign attributes (#596)